### PR TITLE
feat: check if the validator address belongs to the validator committee

### LINKED ISSUE #8172: [Task]: check if the validator address belongs to the validator committee
### Task description

check if the validator address belongs to the validator committee

### PR DESCRIPTION
**Pull request by cpl121**
Closes #8172 

## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

...

## Changelog

```
- Please write a list of changes
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/views/dashboard/delegation/Delegation.svelte [hunk 1]
```diff
@@ -180,7 +183,19 @@
                         isCopyable: true,
                         clearBoxPadding: true,
                         clearBackground: true,
-                        classes: 'text-gray-600 dark:text-white text-xs font-medium',
+                        classes: 'flex flex-row items-center text-gray-600 dark:text-white text-xs font-medium gap-2',
+                    },
+                    slot: {
+                        component: PingingBadge,
```
Comment (evavirseda) [inline packages/desktop/views/dashboard/delegation/Delegation.svelte:189]: i'm not sure about using this component here... maybe we should ask the rest of the team :pray:  wdyt?

Comment (cpl121) [reply packages/desktop/views/dashboard/delegation/Delegation.svelte:189]: I see it well, ask the rest of the team if you want

**Comment by cpl121** [comment]
> I am getting `logAndNotifyError.ts:15 cannot deserialize AccountAddress: missing field accountId at line 1 column 54` not sure if whether I am using an old SDK or what

I can see it well with the latest version of the sdk
Captura de pantalla de 2024-03-19 14-21-04

**Review by evavirseda — APPROVED** [review]
looks good!

**Review by begonaalvarezd — APPROVED** [review]

**Pull request merged**